config/snowflake.spc

```diff
@@ -16,6 +16,10 @@ connection "snowflake" {
   # If not specified, the default role for the user will be used
   # role = "ACCOUNTADMIN"
 
+  # Specifies the sSnowflake warehouse to use for executing snowflake queries (Required for schemata, session and login_history tables)
```

Comment:
```suggestion
  # Specifies the Snowflake warehouse to use for executing snowflake queries
```

We can leave the table names out in the config notes, instead adding a note in each table's doc